### ISSUE TITLE
Ambiguous method generation

### ISSUE THREAD

**Issue by xaviergxf** [RSuter/NSwag#1145]
[swagger.json.txt](https://github.com/RSuter/NSwag/files/1638547/swagger.json.txt)
The dot net cli nswag version 11.12.16 generated an ambiguous c# client method AmloDecisionGetAsync. Is there any way to change the template and add just one method with cancellationtoken=default(CancellationToken)?

Thanks

**Comment by RSuter** [comment]
You can overwrite one of the templates, see https://github.com/RSuter/NSwag/wiki/Templates

But you should check why the ambiguous method is generated in the first place. Is the operationId used twice?

**Comment by xaviergxf** [comment]
This is precisely the problem... There's no operationId only paths, that's probably the reason why it cannot generate unique methods.

**Comment by RSuter** [comment]
Yes, if you only have the paths, then it often generates duplicate methods.. i think there is a new generator which also adds the http method to the method to aboid this... otherwise you have to add operationids (controller_method) or implement an own operation name generator
